homeassistant/components/calendar/todoist.py

```diff
@@ -186,7 +192,16 @@ def handle_new_task(call):
         if PRIORITY in call.data:
             item.update(priority=call.data[PRIORITY])
 
+        if DATE_STRING in call.data:
+            item.update(date_string=call.data[DATE_STRING])
+
+        if DATE_LANG in call.data:
+            item.update(date_lang=call.data[DATE_LANG])
+
         if DUE_DATE in call.data:
+            # Make sure only DUE_DATE is set
```

Comment:
I think this should at least issue a warning here, if it's not already caught by service validation (see above).